packages/manager/apps/vrack-services/src/pages/index.tsx

```diff
@@ -47,10 +45,14 @@ export default function ListingPage() {
     );
   }
 
-  if (error || data?.status !== 200) {
+  if (data?.data.length === 0) {
     return <Navigate to="onboarding" />;
   }
 
+  if (error || data?.status !== 200) {
```

Comment:
```suggestion
  if (error) {
```
just this condition is enough now with the Promise.reject(error) in client.ts

Reply:
fixed on another PR